### ISSUE TITLE
Size 1/3 and 2/3 with small Display Resolution

### ISSUE THREAD

**Issue by dimonnwc3** [eczarny/spectacle#394]
I have a MacBook retina 13. OS X 10.10.3
Resolution looks like a 1440 X 900, 1680 X 1050, and an external monitor - work perfectly!
But the default resolution 1280 X 800, has a problem with many applications.

For example, change the size to 1/2, 1/3, 2/3 in a texteditor works fine. Applications such as the Finder, Safari, Notes, Reminders can not be reduced to 1/3, as their smallest possible size has more than 1/3 of the screen. Size 2/3 is activated only after the size 1/3 but before the size 1/3 is not activated, 2/3 is also not possible.

I propose to small resolutions, instead of the size 1/3 to make the smallest possible size of the application, but rather the size of 2/3 make the difference between the available sizes on the screen and the smallest possible size of the application.

Video recording bug in the attachment.

https://www.dropbox.com/sc/0yqpab0sadzrcc9/AADEZKsp2zqhSHxdkAVkThz1a

**Comment by eczarny** [comment]
Closing as a duplicate of #388.
